openupgrade_scripts/scripts/analytic/17.0.1.2/pre-migration.py

```diff
@@ -22,6 +25,30 @@ def _fill_config_parameter_analytic_project_plan(env):
         [("module", "=", "analytic"), ("name", "=", "analytic_plan_projects")]
     )
     plan_id = imd.res_id
+    if not plan_id and openupgrade.column_exists(
+        env.cr, "res_company", "internal_project_id"
```

Comment:
that's the id of a project, I think you'll want https://github.com/odoo/odoo/blob/16.0/addons/project/models/company.py#L9 here?